src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -103,6 +126,18 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
                 updatedEmail, updatedAddress, updatedTags);
     }
 
+    /**
+     * Creates and returns a {@code Activity} with the details of {@code activityToEdit}
+     * edited with {@code editActivityDescriptor}.
+     */
+    private static Activity createEditedActivity(Activity activityToEdit,
+                                                 EditActivityDescriptor editActivityDescriptor) {
+        assert activityToEdit != null;
```

Comment:
`requireNonNull` may be more suitable for this.

Reply:
i simply stole this from EditPersonDescriptor